DatadogSDKBridge.podspec.src

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '11.0'
   s.source_files = 'DatadogSDKBridge/Classes/**/*'
-  s.dependency 'DatadogSDK', '~> 1.5.1'
+  s.dependency 'DatadogSDK', '~> 1.6.0'
```

Comment:
This should stay aligned with `DatadogSDKBridge.podspec` value (`1.6.0-alpha1`). The way it works: we update `.podspec.src` file and `make bump` command creates the `.podspec`:
https://github.com/DataDog/dd-bridge-ios/blob/063b5809f635d44dab59ca03a3df5deb7869da55/Makefile#L16

Reply:
I made it on purpose, because not sure if we want to release `dd-bridge-ios` with alpha-version dependency. do you think it will be safe to merge it with alpha?

Reply:
I see 👌. Indeed, we don't have clear rule for such case so it may work as you suggest 👍. // cc @buranmert